src/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -190,11 +190,7 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
                 data-testid="logoutDropdown"
               >
                 {data?.organizations[0].image ? (
-                  <img
-                    src={data?.organizations[0].image}
-                    className={styles.roundedcircle}
```

Comment:
Why did you remove this image? Won't we need it for wider screens?

Reply:
I don't think we need the image on the right side of the navbar even for the wider screen because we already have it on the left side, the repetition is unnecessary and it doesn't really serve any purpose, replacing that image with with a span element that says "more" serves two purposes.

1. It tells the user that there is more to explore on that side of the navbar.
2. And it eliminates the repetition of same image.

![navbar wide screen](https://user-images.githubusercontent.com/104039356/226122644-9aed7845-b531-46ed-8426-96a0f29e0afa.jpg)